common/xrdp_client_info.h

```diff
@@ -241,6 +241,11 @@ struct xrdp_client_info
 
     int pad1; /* unused; unicode_input_state */
     enum xrdp_capture_code capture_code;
+
+    /* xorgxrdp: frame capture interval (milliseconds) */
+    int rfx_frame_interval;
+    int h264_frame_interval;
+    int normal_frame_interval;
```

Comment:
These values need to be above the following comment in this file:-

```c
/* ==================================================================== */
    /* Private to xrdp below this line */
    /* ==================================================================== */

```

Reply:
Indeed.